src/liger_kernel/ops/geglu.py

```diff
@@ -11,7 +11,7 @@
 )
 
 if compare_version("triton", operator.ge, "3.0.0"):
-    from triton.language.extra.libdevice import tanh
+    from triton.language.extra.cuda.libdevice import tanh
```

Comment:
i have a bit of concern that this can mean removing AMD support. Maybe we can do something like below?
```python
if compare_version("triton", operator.ge, "3.0.0"):
    try:
        # typical import path with dispatch available
        from triton.language.extra.libdevice import tanh
    except ModuleNotFoundError:
        # for working with NGC containers
        from triton.language.extra.cuda.libdevice import tanh
else:
    from triton.language.math import tanh
```

@lancerts it would be super helpful if you could help test it out on your personal PC. Getting NGC container run on our internal env might take some bit of time :/ Thanks!

Reply:
nvm i think it's just triton changing their import path again.. this is the PR from last week https://github.com/triton-lang/triton/pull/4539/files

Reply:
```
root@f07c568e98bf:/usr/local/lib/python3.10/dist-packages/triton# ll language/extra/
total 20
drwxr-xr-x 4 root root 4096 May  3 02:49 ./
drwxr-xr-x 4 root root 4096 May  3 02:49 ../
-rw-r--r-- 1 root root   39 May  3 02:49 __init__.py
drwxr-xr-x 2 root root 4096 May  3 02:49 __pycache__/
drwxr-xr-x 3 root root 4096 May  3 02:49 cuda/
```
exec into `nvcr.io/nvidia/pytorch:24.05-py3` to check the code, looks like indeed they have some modification over there. I think above suggested fix^ should work. @arvindsun Would you mind help making corresponding change to geglu and rmsnorm? Thanks a lot!